.github/workflows/integration-tests.yml

```diff
@@ -107,6 +73,29 @@ jobs:
             ccip_changes:
               - '**/*ccip*'
               - '**/*ccip*/**'
+      - name: Filter not ignored changes
+        uses: ./.github/actions/filter-changes
+        id: filter
+        with:
+          # Filter files that match **every** pattern in this list
+          every-filter: |
+            - '**'                     # Initially include all files
+            - '!**/*.md'               # Exclude all markdown files (lowercase)
+            - '!**/*.MD'               # Exclude all markdown files (uppercase)
+            - '!**/.github/**'         # Exclude all changes under the .github directory
+            # Exclude all Go test files, except those explicitly included in "any-filter"
+            - '!**/*_test.go'
+            - '!testdata/**'           # Exclude all testdata directories
+            - '!tools/**'              # Exclude all tools directories
```

Comment:
We use tools to build the chainlink image, which is what is being put under test. Any change to our build tooling should be subject to being tested. We've already had failures in the past related to build tooling such as `arm64` builds having linker failures, which would only be caught if they were executed.